jit-service/src/main/java/org/dcsa/jit/service/OperationsEventService.java

```diff
@@ -29,14 +44,45 @@ public static class OperationsEventFilters {
     String unLocationCode;
     String facilitySMDGCode;
     String operationsEventTypeCode;
-    String eventCreatedDateTime;
+    List<ParsedQueryParameter<OffsetDateTime>> eventCreatedDateTime;
     String sort;
     Integer limit;
     String cursor;
     String apiVersion;
   }
 
-  public List<OperationsEventTO> findAll(OperationsEventFilters operationsEventFilters) {
-    return eventRepository.findAll().stream().map(operationsEventMapper::toTO).toList();
+  public List<OperationsEventTO> findAll(
+      HttpServletRequest request,
+      HttpServletResponse response,
+      final OperationsEventFilters requestFilters) {
+
+    Cursor cursor =
+        paginator.parseRequest(
```

Comment:
Still belongs in the controller. The service should not know about http/web stuff

Reply:
Moved